docs/development/core/development-functional-tests.asciidoc

```diff
@@ -133,6 +141,35 @@ The `FunctionalTestRunner`'s primary purpose is to execute test files. These fil
 **Test Suite**:::
 A test suite is a collection of tests defined by calling `describe()`, and then populated with tests and setup/teardown hooks by calling `it()`, `before()`, `beforeEach()`, etc. Every test file must define only one top level test suite, and test suites can have as many nested test suites as they like.
 
+**Tags**:::
+We use tags to group functional tests:
+* `ciGroup{id}` tag to assign test suite to specific CI worker
+* `skipCloud` and `skipFirefox` to exclude test suite from running on Cloud or Firefox browser
+* `smoke` to group tests that we run both on Chrome and Firefox
+Tags are supported for `describe()` function only.
+
+**Cross-browser testing**:::
+On CI all the functional tests are executed in Chrome by default. To run tests also on Firefox `smoke` tag should be assigned:
+
+["source","js"]
+-----------
+// on CI test suite will be run twice: in Chrome and Firefox
+describe('My Cross-browser Test Suite', function () {
+  this.tags('smoke');
+
+  it('My First Test');
+}
+-----------
+
+If the tests are not applicable for run in Firefox, `skipFirefox` tag must be assigned.
```

Comment:
How about:

If the tests do not apply to Firefox, assign the `skipFirefox` tag.